Examples/Swift/InsetMapExample.swift

```diff
@@ -46,8 +46,13 @@ class InsetMapExample_Swift: UIViewController, MGLMapViewDelegate {
         miniMapview.attributionButton.tintColor = UIColor.clear
         miniMapview.layer.borderColor = UIColor.black.cgColor
         miniMapview.layer.borderWidth = 1
+        
+        /**
+         Setting mini map view zoom level to 2 zoom levels below the main map view to showcase
+         one use case for an inset map.
```

Comment:
>showcase one use case for an inset map

What is the use case? Let's dive in a little deeper here. From a user's perspective, why would it make sense for the mini inset map to show a more zoomed out map?

Reply:
It can be used either to display a broader geographical frame of reference, or display more detail within a particular area of the map.